client/post-editor/index.js

```diff
@@ -14,6 +14,7 @@ module.exports = function() {
 	page( '/post', controller.pressThis, sitesController.siteSelection, sitesController.sites );
 	page( '/post/new', () => page.redirect( '/post' ) ); // redirect from beep-beep-boop
 	page( '/post/:site?/:post?', sitesController.siteSelection, sitesController.fetchJetpackSettings, controller.post );
+	page( '/post/revision/:site?/:post?', sitesController.siteSelection, sitesController.fetchJetpackSettings, controller.revision );
```

Comment:
I'd prefer to see `/post/:site/:post/revisions` (note: `revisions` comes last, and there are no `?` or optional parameters).  I don't think we can ever reach this point before selecting a site and a post.

Also, it doesn't have to be done in this PR, but it's worth thinking about whether the approach we're using will handle revisions for post types other than `post` (`page` and custom post types).